### PR TITLE
My 1st js  hw

### PR DESCRIPTION
**Pull request by AntonBakaiev**

### CODE REVIEW

File: students/AntonBakaiev/hw-1/app.js [hunk 1]
```diff
@@ -0,0 +1 @@
+alert('Warning!')
```
Comment (Temu4) [inline students/AntonBakaiev/hw-1/app.js:1]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

File: students/AntonBakaiev/hw-1/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <title>hw-1</title>
+</head>
+<body>
+    <p>I'm just a text</p>
+    <p>The second one</p>
+    <script src="app.js"></script>
+</body>
+
+
+</html>
```
Comment (Temu4) [inline students/AntonBakaiev/hw-1/index.html:20]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

File: students/temu4/README.md [hunk 3]
```diff
@@ -1 +0,0 @@
-test repo file
```
Comment (Temu4) [inline students/temu4/README.md:1]: Pls don't change other files

File: students/AntonBakaiev/hw-1/index.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+    <link rel="stylesheet" href="style.css">
+    <script>
```
Comment (Temu4) [inline students/AntonBakaiev/hw-1/index.html:8]: One of the good practices is not to include scripts in the `head` section of a document. Better place to it at the bottom of the `body` section.

**Review by Temu4 — CHANGES_REQUESTED** [review]
LGTM!
Needs some changes:

File: students/AntonBakaiev/hw-1/index.html (hunk 4 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-1/index.html]: fixed

**Review by Temu4 — APPROVED** [review]

File: students/temu4/README.md (hunk 3 above)
Comment (Temu4) [reply students/temu4/README.md:1]: @AntonBakaiev please revert changes for this file

**Review by Temu4 — CHANGES_REQUESTED** [review]

**Review by Temu4 — APPROVED** [review]

**Pull request merged**